src/handlers/decision.rs

```diff
@@ -0,0 +1,340 @@
+                                    resolution: resolution,
+                                }),
+                            );
+                            history.insert(user.login.clone(), Vec::new());
+
+                            // Initialize issue decision state
+                            insert_issue_decision_state(
+                                &db,
+                                &issue.number,
+                                &user.login,
+                                &start_date,
+                                &end_date,
+                                &current,
+                                &history,
+                                &reversibility,
+                                &resolution,
+                            )
+                            .await?;
+
+                            // TO DO -- Do not insert this job until we support more votes
+                            // let metadata = serde_json::value::to_value(DecisionProcessActionMetadata {
+                            //     message: "some message".to_string(),
+                            //     get_issue_url: format!("{}/issues/{}", issue.repository().url(), issue.number),
+                            //     status: resolution,
+                            // })
+                            // .unwrap();
+                            // insert_job(
+                            //     &db,
+                            //     &DECISION_PROCESS_JOB_NAME.to_string(),
+                            //     &end_date,
+                            //     &metadata,
+                            // )
+                            // .await?;
+
+                            let comment = build_status_comment(&history, &current)?;
+                            issue
+                                .post_comment(&ctx.github, &comment)
+                                .await
+                                .context("merge vote comment")?;
+
+                            Ok(())
+                        }
+                        _ => {
+                            let cmnt =
+                                ErrorComment::new(&issue, "Failed to resolve to a known team.");
+                            cmnt.post(&ctx.github).await?;
+
+                            Ok(())
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+fn build_status_comment(
+    history: &BTreeMap<String, Vec<UserStatus>>,
+    current: &BTreeMap<String, Option<UserStatus>>,
+) -> anyhow::Result<String> {
+    let mut comment = "| Team member | State |\n|-------------|-------|".to_owned();
+    for (user, status) in current {
+        let mut user_statuses = format!("\n| @{} |", user);
+
+        // previous stasuses
+        match history.get(user) {
+            Some(statuses) => {
+                for status in statuses {
+                    let status_item = format!(" ~~{}~~ ", status.resolution);
```

Comment:
If we can link these to the user event (comment) that caused the status change that would be nice.